### ISSUE TITLE
Streamable JSON encoding for timeseries.

### ISSUE THREAD

**Issue by dblodgett-usgs** [opengeospatial/ogcapi-environmental-data-retrieval#343]
The current recommended encodings do not lend themselves to streaming -- we should seek out a streamable encoding for timeseries. Could flesh this out much more, just leaving as a note for the future.

**Comment by tomkralidis** [comment]
Related: https://datatracker.ietf.org/doc/html/rfc7464

**Comment by dblodgett-usgs** [comment]
Yes. Indeed. I'd be curious to see how you would work with metadata surrounding a stream like that. Looks like it's really designed for logging.

**Comment by lvdbrink** [comment]
Just a tip, without knowing if it might be useful: CityJSON 1.1 does streaming, you might check out their solution of [streaming with cityjsonfeature](https://www.cityjson.org/specs/1.1.0/#text-sequences-and-streaming-with-cityjsonfeature). They use [JSON lines text streams](https://jsonlines.org/). I don't know its relationship to rfc7464.

**Comment by chris-little** [comment]
This is a good idea, but there are a lot of issues around streaming JSON, and whether it is applcable to any or all EDR queries. Needs to be linked to Pub/Sub requireement, and perhaps a different service endpoint may be appropriate.  Tagged as V1.3 at the earliest.

**Comment by ghobona** [comment]
The adjective "streamable" might be problematic because there is a company called [Streamable](https://streamable.com) that has its own [JSON encoding and API](https://support.streamable.com/article/46-streamable-api). 

This was pointed out in the Vector Tiles Pilot a few years ago.
